lib/integration/common.ts

```diff
@@ -1,4 +1,62 @@
 import { DateTime } from "luxon";
 import { TIME_ZONE } from "../../config/config";
+import { RezervoSchedule, RezervoWeekSchedule } from "../../types/rezervo";
+import { createClassPopularityIndex } from "../popularity";
 
 export const calculateMondayOffset = () => DateTime.now().setZone(TIME_ZONE).weekday - 1;
+
+export const getDateTime = (date: string): DateTime => DateTime.fromISO(date, { zone: TIME_ZONE, locale: "no" });
+
+export const getCapitalizedWeekday = (date: DateTime): string =>
+    `${date.weekdayLong![0]!.toUpperCase()}${date.weekdayLong!.slice(1)}`;
+
+export function uniteSchedules(schedules: RezervoSchedule[]): RezervoSchedule {
```

Comment:
If we need this function, maybe call it `joinSchedules` to sound less dramatic?

Reply:
I decided to inline this